sdk/servicebus/Azure.Messaging.ServiceBus/tests/ProcessorLiveTests.cs

```diff
@@ -83,7 +91,14 @@ public async Task Receive_StopProcessing(int numThreads)
                 await using var client = new ServiceBusClient(TestEnvironment.ServiceBusConnectionString);
                 ServiceBusSender sender = client.GetSender(scope.QueueName);
                 int numMessages = 50;
-                await sender.SendBatchAsync(GetMessages(numMessages));
+                IEnumerable<ServiceBusMessage> messages = GetMessages(numMessages);
+                using ServiceBusMessageBatch batch = await sender.CreateBatchAsync();
+
+                foreach (ServiceBusMessage message in messages)
+                {
+                    Assert.That(() => batch.TryAdd(message), Is.True, "A message was rejected by the batch; all messages should be accepted.");
```

Comment:
Any chance we can incorporate this into a helper method, similar to GetMessages, that returns a message batch? It would be nice to avoid repeating this in every test if possible.